src/Patch/Node.js

```diff
@@ -137,14 +139,15 @@ export default function(internals) {
      * @return {!Node}
      */
     function(nodeToInsert, nodeToRemove) {
+      const needsConnectTree = Utilities.isConnected(this);
```

Comment:
Can we put this one back into the two places it was called given that it's only called once per path through the function?